airflow/www/static/js/dag/details/dagRun/MarkRunAs.tsx

```diff
@@ -58,42 +65,115 @@ const MarkRunAs = ({ runId, state, ...otherProps }: Props) => {
     markSuccess({ confirmed: true });
   };
 
+  useEffect(() => {
+    const storedSuccessValue = localStorage.getItem("doNotShowAgainSuccess");
+    const storedFailedValue = localStorage.getItem("doNotShowAgainFailed");
```

Comment:
Do we need separate do not show again? I think its ok to just have one.

Reply:
So does that mean neither of the Mark Run As dialog boxes (Success/Failed) should appear again on clicking the do not show again checkbox in any of the two?